modules/local/generate_images.nf

```diff
@@ -0,0 +1,33 @@
+process GENERATE_IMAGES {
+    tag "$meta.id"
+    label 'process_single'
+
+    if (params.enable_conda) {
+        exit 1, "Conda environments cannot be used when using the GENERATE_IMAGES module. Please use docker or singularity containers."
+    }
+    container 'genomehubs/blobtoolkit:4.0.7'
+
+    input:
+    tuple val(meta), path(blobdir)
+
+    output:
+    tuple val(meta), path("${blobdir}/cumulative.png") , emit: png
+    path "versions.yml"                                , emit: versions
+
+    when:
+    task.ext.when == null || task.ext.when
+
+    script:
+    def prefix = task.ext.prefix ?: "${meta.id}"
+    def args = task.ext.args ?: ''
+    """
+    cov=`find -L ./ -name "*_cov.json"`
+    btk pipeline generate-static-images \\
+        --blobdir ${blobdir} \$cov \\
```

Comment:
I don't see space for json files on the command-line, cf
```
$ btk pipeline generate-static-images --help
Generate static image files.

Usage:
  btk pipeline generate-static-images --blobdir STRING [--coverage] [--host STRING]
                         [--ports RANGE] [--timeout INT] [--path PATH]

Options:
  --blobdir STRING  BlobDir to generate static views for.
  --coverage        Flag to indicate if coverage data are available and blob views should be generated. [Default False].
  --host STRING     Host name. [Default: http://localhost]
  --path STRING     Path to directory containing BlobDir. [Default: .]
  --ports RANGE     Range of ports available for hosting BlobTools Viewer/API. [Default: 8000-8099] = str(snakemake.params.ports)
  --timeout INT     Maximum time (in seconds) to wait for images. [Default: 30]
```



```suggestion
    btk pipeline generate-static-images \\
        --blobdir ${blobdir} \\
```

But even if it were accepting json files, there are some issues:
- blobdir is staged as symbolic link, and find doesn't traverse symlinks, so the find command would need to look into blobdir explicitly
- there are no _cov.json files in the example blobdir (there is a _cov.json.gz instead)

Finally
- the output file `cumulative.png` is already in blobdir, so we can't actually test that it is generated

Reply:
I checked the [script](https://github.com/blobtoolkit/blobtoolkit/blob/main/src/blobtoolkit-pipeline/src/blobtoolkit-snakefiles/rules/generate_images.smk) again and the error I made was that `cov` is actually a flag and not a list of files in BlobDir, so it should be omitted  and a flag should be used when those files are available instead.